feature/notification/build.gradle.kts

```diff
@@ -0,0 +1,16 @@
+plugins {
+    alias(libs.plugins.mifospay.android.feature)
+    alias(libs.plugins.mifospay.android.library.compose)
+}
+
+android {
+    namespace = "org.mifospay.notification"
+}
+
+dependencies {
+    implementation(projects.core.data)
+    implementation(libs.androidx.appcompat)
+
+    //we need this for pull to refresh implementation
+    implementation("androidx.compose.material:material:1.6.0")
```

Comment:
we should have this library in designsystem or UI module check and confirm

Reply:
and never use hard coded library, always use from libs

Reply:
Yes I will use libs now